README.md

```diff
@@ -1,18 +1,56 @@
-# soTired
-![build](https://github.com/teamulster2/soTired/actions/workflows/main.yml/badge.svg)
+![pipeline](https://github.com/teamulster2/soTired/actions/workflows/main.yml/badge.svg)
+![license](https://img.shields.io/github/license/teamulster2/soTired)
 
-## Motivation
+<div align="center">
+  <img src="./app/icon/icon.png">
+  <p style="font-size:70px;">
+    soTired
+  </p>
+</div>
 
+## Motivation
 soTired is an application for cognitive fatigue assessment.
-It includes a stand-alone Android app for fatigue detection and an additional part for data management and further analysis.
-The project is structured as client side (stand-alone app) implemented in Dart/Flutter and a server side (data management) written in Golang.
-Besides Android, Dart/Flutter provides the ability to add an iOS, desktop and / or web application from a single codebase.
+It includes a stand-alone Android app for fatigue detection and an additional part for data 
+management and further analysis. The project is structured as client side (stand-alone app) 
+implemented in Dart/Flutter and a server side (data management) written in Golang.
+Besides Android, Dart/Flutter provides the ability to add an iOS, desktop and / or web application 
+from a single codebase.
 
+soTired is a rewrite of the application presented in the 
+[Validation of a Smartphone-Based Approach to In Situ Cognitive Fatigue Assessment](https://mhealth.jmir.org/2017/8/e125) 
+paper by Edward Price, George Moore, Leo Galway and Mark Linden.
 
-soTired is a rewrite of the application presented in the [Validation of a Smartphone-Based Approach to In Situ Cognitive Fatigue Assessment](https://mhealth.jmir.org/2017/8/e125) paper by Edward Price, George Moore, Leo Galway and Mark Linden.
+## How to contribute
+PRs are very welcome! :)  
+You can set up your development environments following the steps below.
 
-## Contributions
+### App development environment setup
+1. Make sure you have flutter (version `2.5.0`) installed and `flutter doctor -v` does not have any 
+errors regarding Android development.
+1. Clone the repository and run `cd soTired/ && make init` to initialize git pre-commit hooks.
+1. Inside the `soTired/` directory run `make flutterPubGet` to install all depending packages.
+
```

Comment:
Perhaps we can add here that we used Android version 9.0

Reply:
Addressed in commit 0765521